tests/features/interactive_steps.feature

```diff
@@ -83,3 +83,20 @@ Scenario: I set text-type values
   And I tap to continue
   # Next page
   Then the question id should be "the end"
+
+@fast @i2
+Scenario: handles settings from Github secrets
+  Given I start the interview at "text_tests"
+  When I set the var first_text_entry to the secret "SECRET_VAR1"
```

Comment:
I don't quite have bandwidth to review everything at the moment, and sorry I didn't see this before. This isn't passing, but not just because the file doesn't exist on the other server.  I think the var name is missing quotes:

```suggestion
  When I set the var "first_text_entry" to the secret "SECRET_VAR1"
```

That might seem out of the blue, but I'm getting it from the tests. In the cucumber output I can see
```
   ? When I set the var first_text_entry to the secret "SECRET_VAR1"
       Undefined. Implement with the following snippet:
```
which means it's not recognizing the cucumber step you're using.